workflows/sarek.nf

```diff
@@ -1122,7 +1122,10 @@ workflow SAREK {
             known_snps_vqsr,
             params.joint_germline,
             params.skip_tools && params.skip_tools.split(',').contains('haplotypecaller_filter'), // true if filtering should be skipped
-            params.sentieon_haplotyper_emit_mode)
+            params.sentieon_haplotyper_emit_mode,
+            params.sentieon_dnascope_emit_mode,
+            params.sentieon_dnascope_pcr_based,
+            params.sentieon_dnascope_model)
```

Comment:
This being a file should be a channel, like we do for fasta for example

Reply:
Ok, thanks for pointing that out. I guess it is important for the correct "staging" of the file, or something like that, right?